dired-sidebar.el

```diff
@@ -740,7 +750,9 @@ Select alternate window using `dired-sidebar-alternate-select-window-function'."
          (progn
            (switch-to-buffer up-name)
            (dired-sidebar-update-state (current-buffer)))
-       (dired-up-directory)
+         (if dired-sidebar-one-instance
+             (find-alternate-file "..")
```

Comment:
Same comment as above. This time it's about dired-up-directory vs using find-alternate-file.

Admittedly, I'm not too against it but this code can be hard to follow already since there's a lot of conditionals at this point.

Reply:
Same reason as above.

> Admittedly, I'm not too against it but this code can be hard to follow already since there's a lot of conditionals at this point.

I'm not sure how to improve this, you could help me after merged.